languages/fsharp/reference/exercise-concepts/bob.md

```diff
@@ -0,0 +1,46 @@
+# Concepts of bob
+
```

Comment:
Could you add a link here to the v2 implementation?

Reply:
Done!